plugins/localfile/localfile.go

```diff
@@ -0,0 +1,57 @@
+package localfile
+
+import (
+	"compress/gzip"
+	"encoding/csv"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/stripe/veneur/plugins"
+	"github.com/stripe/veneur/plugins/s3"
+	"github.com/stripe/veneur/samplers"
+)
+
+var _ plugins.Plugin = &Plugin{}
+
+// Plugin is the LocalFile plugin that we'll use in Veneur
+type Plugin struct {
+	FilePath string
+	Logger   *logrus.Logger
+}
+
+// Delimiter defines what kind of delimiter we'll use in the CSV format -- in this case, we want TSV
+const Delimiter = '\t'
+
+// Flush the metrics from the LocalFilePlugin
+func (p *Plugin) Flush(metrics []samplers.DDMetric, hostname string) error {
+	f, err := os.OpenFile(p.FilePath, os.O_RDWR|os.O_APPEND|os.O_CREATE, os.ModePerm)
```

Comment:
Agreed on not wanting to encourage people to store archives of their metrics on a single host (it's possible there's a use case for that and we'll get a PR for it, but let's cross that bridge if we get there).

I think our reasoning there would be a bit clearer though if we explained in the README for the localfile what the intended purpose is (ie, temporary caching, essentially).